core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/ItemSelectionInputCheckerServiceSpec.js

```diff
@@ -106,6 +106,30 @@ describe('Checkbox input checker service', function() {
       },
       correct: false
     }];
-    expect(mcics.isValid(customizationArgs, answerGroupsTrue)).toBe(false);
+    expect(isics.isValid(customizationArgs, answerGroupsTrue)).toBe(false);
+  });
+
+  it('should return false for rule sets larger than number of choices', 
```

Comment:
It's not just a case of "larger", right? We should check that if a rule set contains something that's not in the customization args choices, the whole thing fails validation. Regardless of whether it's larger/smaller in length.

Reply:
that is true, this one was specifically testing for the larger case. I'll write one for the "not in customization args" case.

Reply:
Altered the test and added a check in the Checker Service.